HPXMLtoOpenStudio/resources/lighting.rb

```diff
@@ -70,18 +70,8 @@ def self.apply(runner, model, epw_file, spaces, lighting_groups, lighting, eri_v
       end
       if interior_sch.nil?
         interior_unavailable_periods = Schedule.get_unavailable_periods(runner, interior_col_name, unavailable_periods)
-        if not lighting.interior_weekday_fractions.nil?
-          interior_sch = MonthWeekdayWeekendSchedule.new(model, interior_obj_name + ' schedule', lighting.interior_weekday_fractions, lighting.interior_weekend_fractions, lighting.interior_monthly_multipliers, Constants.ScheduleTypeLimitsFraction, unavailable_periods: interior_unavailable_periods)
-        else
-          lighting_sch = get_schedule(epw_file)
```

Comment:
Does this whole `get_schedule` location-based approach for determining interior lighting schedules go away / get replaced?

Reply:
Good question. Since the current default is pretty "smart", my initial thought is that perhaps we should leave the lighting default in OS-HPXML and instead just set the lighting schedules in the HPXML files configured by OS-ERI.

Reply:
But use the new set of monthly lighting multipliers in `get_schedule`?

Reply:
What are the new set of monthly lighting multipliers?

Reply:
[New](https://github.com/NREL/OpenStudio-HPXML/blob/ansi_301_2022_lighting_appl/HPXMLtoOpenStudio/resources/schedules.rb#L865-L867) vs [Old](https://github.com/NREL/OpenStudio-HPXML/blob/ansi_301_2022/HPXMLtoOpenStudio/resources/schedules.rb#L841-L843).